types/read_config.go

```diff
@@ -71,4 +82,5 @@ type BootstrapConfig struct {
 	EnableFunctionReadinessProbe bool
 	ReadTimeout                  time.Duration
 	WriteTimeout                 time.Duration
+	ImagePullPolicy              string
```

Comment:
Why not make the `ImagePullPolicy` of type `apiv1.PullPolicy`? https://github.com/golang/build/blob/master/kubernetes/api/types.go#L863

Reply:
I left it generic here since `deploy.go` is where the deployment is being constructed (and the k8s api is already imported there); and because read_config (so far) is only using native go packages.  I could move it here but that is probably not necessary unless/until it's being reused by more than one handler.  If you guys feel strongly the logic should move here let me know.